dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -174,14 +174,25 @@ public Experiment save(final Experiment experiment, final User user) throws
 
             addConditionIfIsNeed(goals,
                     APILocator.getHTMLPageAssetAPI().fromContentlet(pageAsContent), builder);
-
         }
 
         if(experiment.targetingConditions().isPresent()) {
             saveTargetingConditions(experiment, user);
         }
 
-        final Experiment experimentToSave = builder.build();
+        Experiment experimentToSave = builder.build();
+
+        Optional<Experiment> existingExperiment = find(experimentToSave.id().get(), user);
+
+        if(experimentToSave.status() != RUNNING &&
```

Comment:
I know this code come from other PR.
Maybe we should extract this to a function returning a boolean for the first 3 evaluations:
```
static List<STATUS> SOME_NAME = List.of(RUNNING, ENDED, ARCHIVED);
...
boolean hasNotStarted(List<STATUS> statuses, Experiment experiment) {
  return !SOME_NAME.contains(experiment.status())
}
```

Reply:
Simplified the condition and extracted second part to a private method